Userland/Utilities/run-tests.cpp

```diff
@@ -81,6 +81,11 @@ Vector<ByteString> TestRunner::get_test_paths() const
     Test::iterate_directory_recursively(m_test_root, [&](ByteString const& file_path) {
         if (access(file_path.characters(), R_OK | X_OK) != 0)
             return;
+        if (file_path.ends_with(".so"sv) || file_path.ends_with(".elf"sv)) {
```

Comment:
Is there an entry in the run-tests config file (/home/anon/.config/Tests.ini ?) that already covers the .so files?

Reply:
we should add them to `NotTestsPattern=^.*(txt|frm|inc)$`